spyder/utils/tests/test_pyenv.py

```diff
@@ -15,6 +15,10 @@
 from spyder.utils.pyenv import get_list_pyenv_envs, get_list_pyenv_envs_cache
 
 
+PYENV_MISSING = (len(get_list_pyenv_envs()) == 0)
```

Comment:
Please change this line to

```python
if not find_program('pyenv'):
    pytest.skip()
```

and remove the `skipif` you added in each test below.

`find_program` is available in `spyder.utils.programs`.

Reply:
All three requests fulfilled!